src/phoenix/session/session.py

```diff
@@ -594,6 +621,9 @@ def close_app() -> None:
     _session.end()
     _session = None
     logger.info("Session closed")
+    if delete_data:
+        logger.info("Deleting all data")
+        delete_all(prompt_before_delete=False)
```

Comment:
Maybe we should default to True
```suggestion
        delete_all(prompt_before_delete=True)
```

Reply:
But in this case they explicitly set the delete_data=True. I think I'm going to leave this for now.